README.md

```diff
@@ -1 +1,13 @@
-# angular-testing-filters-lab
\ No newline at end of file
+# Testing dataset Filters
+
+## Objectives
+
+- Write a unit test to test filter integration
+
+## Instructions
+
+In this repo we've got the same setup as our last lab. We've got a `ContactController` with 150 contacts in it.
+
+Let's test out that we can filter this large dataset.
+
+Write various different tests that search for different aspects of our contacts - for instance, write a test to return all the males. You can also write one to select people from different geographical locations.
```

Comment:
Here, let's tell them specifically what to test for here and what the results should be. Tell them the number of people who should be returned when the search result is 'female' and then have them write the test for it. Do the same thing for geographical locations - tell the user how many people should show up for 'manchester' and have them add the test.